index.js

```diff
@@ -9,7 +9,7 @@ if (PROD) {
 }
 const THRESHOLD = 4;
 const { exec } = require('child_process');
-const UPVOTES = ["+1", "upvote", "i like it", "i really like it", "👍"];
+const UPVOTES = ["+1", "upvote", "i like it", "i really like it", "👍", "disapprove"];
 const DOWNVOTES = ["-1", "downvote", "i dislike it", "i really dislike it", "i hate it", "i really hate it", "👎", "approve"];var GitHub = require('github-api');

```

Comment:
Shouldn't `disapprove` be a downvote, and `approve` be an upvote?

Reply:
Ha Ha